client/my-sites/subscribers/main.tsx

```diff
@@ -60,29 +61,39 @@ export const Subscribers = ( { page, pageChanged }: SubscribersProps ) => {
 	}
 
 	return (
-		<Main wideLayout className="subscribers">
-			<DocumentHead title={ translate( 'Subscribers' ) } />
-			<FixedNavigationHeader navigationItems={ navigationItems }></FixedNavigationHeader>
+		<div className="subscribers">
+			<Main wideLayout className="subscribers__main">
```

Comment:
I wonder how strictly the following guidance should be followed (as the PR adds one wrapper `<div>`). 🤔 

> The first component that is rendered for any section in #primary should be called section/main.jsx. That component should make use of components/main for it's wrapper.

(from [client/my-sites/README.md](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/README.md))

For instance, the Plugins section (`client/my-sites/plugins/main.jsx`) doesn't follow the guidance and Themes section doesn't have the `main.jsx` file at all.

Do you have any thoughts on this @yansern?